manifests/init.pp

```diff
@@ -25,11 +25,8 @@
   $conf_d_dir= "${conf_dir}/conf.d"
   $conn_dir = "${conf_dir}/system-connections"
 
-  require nm::install
-  require nm::service
-
-  Class['nm::install']
-  -> Class['nm::service']
```

Comment:
Why does this relationship need to be changed?

Reply:
As there is actually no need, I have rolled back this change.